packages/ui/src/theme/tokens/components/collection.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  BackgroundColorValue,
+  ColorValue,
+  DesignToken,
+} from '../types/designToken';
+
+interface PaginationTokens {
+  current: {
+    color: DesignToken<ColorValue>;
+    backgroundColor: DesignToken<BackgroundColorValue>;
+  };
+  button: {
+    color: DesignToken<ColorValue>;
+    _hover: {
+      backgroundColor: DesignToken<BackgroundColorValue>;
+      color: DesignToken<ColorValue>;
+    };
+    _disabled: {
+      color: DesignToken<ColorValue>;
+    };
```

Comment:
Should these be `StateTokens` type? Also should we add `_focus` and `_active` states too?

Reply:
I've updated these to use the hover and disabled types declared in the pagination tokens however I did not add the focus and active states because they don't exist on the pagination component.